DfE.FindInformationAcademiesTrusts/Pages/Trusts/Ofsted/_BeforeOrAfterTag.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@model bool
```

Comment:
For readability and to enable extra states (like not inspected) we could use an enum instead of a bool to store the before/after state

```csharp
public enum BeforeOrAfterJoining
{
    Before,
    After
}
```